src/CommunityToolkit/Xamarin.CommunityToolkit/Views/BadgeView/BadgeView.shared.cs

```diff
@@ -1,366 +1,367 @@
-				var size = Math.Max(BadgeText.Height, BadgeText.Width) + Padding;
-				BadgeIndicatorBackground.HeightRequest = size;
-				var margins = GetMargins(size);
-				containerMargin = margins.Item1;
-				contentMargin = margins.Item2;
-			}
-
-			BadgeIndicatorContainer.Margin = containerMargin;
-			BadgeContent.Margin = contentMargin;
-			placementDone = true;
-		}
-
-		Tuple<Thickness, Thickness> GetMargins(double size)
-		{
-			double verticalMargin;
-			double horizontalMargin;
-			var containerMargin = new Thickness(0);
-			var contentMargin = new Thickness(0);
-			switch (BadgePosition)
-			{
-				case BadgePosition.TopRight:
-					verticalMargin = size / 2;
-					horizontalMargin = BadgeContent.Width - verticalMargin;
-					containerMargin = new Thickness(horizontalMargin, 0, 0, 0);
-					contentMargin = new Thickness(0, verticalMargin, verticalMargin, 0);
-					break;
-				case BadgePosition.TopLeft:
-					verticalMargin = size / 2;
-					containerMargin = new Thickness(0, 0, 0, 0);
-					contentMargin = new Thickness(verticalMargin, verticalMargin, 0, 0);
-					break;
-				case BadgePosition.BottomLeft:
-					verticalMargin = size / 2;
-					var bottomLeftverticalMargin = BadgeContent.Height - verticalMargin;
-					containerMargin = new Thickness(0, bottomLeftverticalMargin, 0, 0);
-					contentMargin = new Thickness(verticalMargin, 0, 0, 0);
-					break;
-				case BadgePosition.BottomRight:
-					verticalMargin = size / 2;
-					var bottomRightverticalMargin = BadgeContent.Height - verticalMargin;
-					horizontalMargin = BadgeContent.Width - verticalMargin;
-					containerMargin = new Thickness(horizontalMargin, bottomRightverticalMargin, 0, 0);
-					contentMargin = new Thickness(0, 0, verticalMargin, 0);
-					break;
-			}
-			return new Tuple<Thickness, Thickness>(containerMargin, contentMargin);
-		}
-
-		async Task UpdateVisibilityAsync()
-		{
-			if (BadgeIndicatorBackground == null)
-				return;
-
-			var badgeText = BadgeText.Text;
-
-			if (string.IsNullOrEmpty(badgeText))
-			{
-				BadgeIndicatorBackground.IsVisible = false;
-				return;
-			}
-
-			var badgeIsVisible = !AutoHide || !badgeText.Trim().Equals("0");
-			BadgeIndicatorBackground.IsVisible = badgeIsVisible;
-			
-			if (IsAnimated)
-			{
-				if (badgeIsVisible == isVisible)
-					return;
-
-				if (badgeIsVisible)
-				{
-					BadgeIndicatorContainer.IsVisible = true;
-					await BadgeAnimation.OnAppearing(BadgeIndicatorContainer);
-				}
-				else
-				{
-					await BadgeAnimation.OnDisappering(BadgeIndicatorContainer);
-					BadgeIndicatorContainer.IsVisible = false;
-				}
-
-				isVisible = badgeIsVisible;
-			}
-			else
-				BadgeIndicatorContainer.IsVisible = badgeIsVisible;
-		}
-
-		void OnBadgeTextSizeChanged(object sender, EventArgs e)
-			=> UpdateBadgeViewPlacement(true);
-
-		void OnBadgeIndicatorContainerPropertyChanged(object sender, PropertyChangedEventArgs e)
-			=> UpdateBadgeViewPlacement(true);
-	}
+    [ContentProperty(nameof(Content))]
```

Comment:
Hmm, looks like tabs were replaced by spaces for some reason

Reply:
The bot can fix that